app/views/mods_panel.py

```diff
@@ -1917,9 +1943,10 @@ def recreate_mod_list_and_sort(
             None
         """
         # TODO: Fix this or just get rid of it
-        sorted_uuids = uuids
+        # sorted_uuids = uuids
         # if key != ModsPanelSortKey.NOKEY:
         #    sorted_uuids = sorted(uuids, key=key)
+        sorted_uuids = sort_uuids(uuids, key = key)
```

Comment:
Needs to go through formatting via Ruff.

Reply:
Hi! I forgot to reply to this one. I used Ruff as formatter in vscode, I'm not sure whether this is the correct way of doing it. Could you please help check it with ruff for this commit?

Reply:
Looks correct in latest commit!